qiita_pet/templates/study_description_templates/prep_template_panel.html

```diff
@@ -41,7 +41,10 @@ <h4 class="modal-title" id="myModalLabel">Choose preprocessing parameters</h4>
 <div class="panel panel-default">
   <div class="panel-heading">
     <h4 class="panel-title">
-      <a data-toggle="collapse" data-parent="#prep-accordion" href="#collapse{{prep_id}}">{{data_type}} (ID: {{prep_id}})</a>
+      <a data-toggle="collapse" data-parent="#prep-accordion" href="#collapse{{prep_id}}">
+        {{data_type}} (ID: {{prep_id}})
+        <button class="close" title="Remove this prep template" type="button" onclick="delete_prep_template('{{data_type}}', {{prep_id}})">&nbsp; ×</button> 
```

Comment:
Nice, I see this is consistent with the way raw datas are deleted. There is just one unintended side-effect. When I click this button and the alert comes up, if I choose Cancel, the prep template accordion is collapsed, not sure if we can bypass that so this doesn't happen?